src/views/Governance/Governance.tsx

```diff
@@ -53,44 +65,65 @@ const Governance: React.FC = () => {
-                    <StyledButtonMain>Action</StyledButtonMain>
-                  </StyledProposalContentInner>
-                </Box>
-                <Spacer size="sm" />
-                {proposals && (
-                  <Surface>
-                    {proposals.map((prop, i) => {
-                      if (i === 0) {
-                        return <ProposalEntry key={prop.hash} prop={prop} onVote={onVote} onRegister={onRegister} />;
-                      } else {
-                        return [<Separator key={"seperator" + i}/>, <ProposalEntry key={prop.hash} prop={prop} onVote={onVote} onRegister={onRegister} />];
-                      }
-                    })}
-                  </Surface>
+        {account
+          ? <Card>
+              <CardTitle text="On-chain Proposals" />
+              <Spacer size="sm" />
+              <CardContent>
+                {proposals ? (
+                  <>
+                    <Box display="grid" alignItems="center" paddingLeft={4} paddingRight={4} paddingBottom={1} row>
+                      <StyledProposalContentInner>
+                        <StyledDescriptionMain>Description</StyledDescriptionMain>
+                        <SeparatorGrid orientation={"vertical"} stretch={true} gridArea={"spacer1"} />
+                        <StyledStateMain>State</StyledStateMain>
+                        <SeparatorGrid orientation={"vertical"} stretch={true} gridArea={"spacer2"} />
+                        <StyledButtonMain>Action</StyledButtonMain>
+                      </StyledProposalContentInner>
+                    </Box>
+                    <Spacer size="sm" />
+                    {proposals && (
+                      <Surface>
+                        {proposals.map((prop, i) => {
+                          if (i === 0) {
+                            return <ProposalEntry key={prop.hash} prop={prop} onVote={onVote} onRegister={onRegister} />;
+                          } else {
+                            return [<Separator key={"seperator" + i}/>, <ProposalEntry key={prop.hash} prop={prop} onVote={onVote} onRegister={onRegister} />];
+                          }
+                        })}
+                      </Surface>
+                    )}
+                  </>
+                ) : (
+                  <YamLoader space={320}></YamLoader>
                 )}
-              </>
-            ) : (
-              <YamLoader space={320}></YamLoader>
-            )}
-          </CardContent>
-        </Card>}
+              </CardContent>
+            </Card>
+          : (
+            <>
+              <Box row justifyContent="center">
+                <Button onClick={handleUnlockWalletClick} text="Unlock wallet to display proposals" variant="secondary" />
+              </Box>
+              <UnlockWalletModal isOpen={unlockModalIsOpen} onDismiss={handleDismissUnlockModal} />
+            </>
```

Comment:
we dont want zero buttons nor we want two buttons, just one button with "Unlock wallet...", and priority here is for the proposals so please keep this one and remove the other